fcc-js/es6/compare-scopes-of-the-var-and-let-keywords/README.md

```diff
@@ -1 +1,75 @@
-# Add challenge details and instructions here
+# Compare Scopes of the var and let Keywords
+
+If you are unfamiliar with `let`, check out [this challenge about the difference between `let` and `var`](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/explore-differences-between-the-var-and-let-keywords).
+
+When you declare a variable with the `var` keyword, it is declared globally, or locally if declared inside a function.
+
+The `let` keyword behaves similarly, but with some extra features. When you declare a variable with the `let` keyword inside a block, statement, or expression, its scope is limited to that block, statement, or expression.
+
+For example:
+
+```
+var numArray = [];
+for (var i = 0; i < 3; i++) {
+  numArray.push(i);
+}
+console.log(numArray);
+console.log(i);
+```
+
+Here the console will display the values `[0, 1, 2]` and `3`.
+
+With the `var` keyword, `i` is declared globally. So when `i++` is executed, it updates the global variable. This code is similar to the following:
+
+```
```

Comment:
same here.

```suggestion
```js
```
```

Reply:
Everything is now showing up as js code snippet sir